### PR TITLE
test: fix lazy loaded strings

### PR DESCRIPTION
**Pull request by JadenSimon**
Module exports were being assigned on file load causing race conditions. Assign them on test start instead.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/test/globalSetup.test.ts [hunk 1]
```diff
@@ -45,6 +46,7 @@ before(async function () {
     ext.init(fakeContext, extWindow.Window.vscode())
     ext.telemetry = service
     await initializeComputeRegion()
+    lazyLoadSamTemplateStrings()
```
Comment (bryceitoc9) [inline src/test/globalSetup.test.ts:49]: Nit: maybe move this out of the global setup and into the before for `samTemplates.test.ts`?

**Review by bryceitoc9 — APPROVED** [review]

File: src/test/globalSetup.test.ts (hunk 1 above)
Comment (JadenSimon) [reply src/test/globalSetup.test.ts:49]: Oh yeah I didn't even realize it was called `SamTemplateStrings` in particular

**Pull request merged**